src/collections/Tags.ts

```diff
@@ -0,0 +1,60 @@
+import { CollectionConfig } from 'payload'
+import { slugField } from '@/fields/slug'
+
+export const Tags: CollectionConfig = {
+  slug: 'tags',
+  labels: {
+    singular: 'Tag',
+    plural: 'Tags',
+  },
+  admin: {
+    useAsTitle: 'name',
+    defaultColumns: ['name', 'slug', 'relatedTo'],
+  },
+  access: {
+    read: () => true,
+  },
+  timestamps: true,
+  fields: [
+    {
+      name: 'name',
+      type: 'text',
+      required: true,
+    },
+    // generate slug from 'name', readOnly, with lock toggle in sidebar
+    ...slugField('name', {}),
+    {
+      name: 'posts',
+      type: 'join',
+      collection: 'posts',
+      on: 'tags',
+      admin: {
+        defaultColumns: ['title'],
+        description: 'Link this tag to one or more Posts',
+        allowCreate: false,
+      },
+    },
+    {
+      name: 'clinics',
+      type: 'join',
+      collection: 'clinics',
+      on: 'tags',
+      admin: {
+        defaultColumns: ['name'],
+        description: 'Link this tag to one or more Clinics',
+        allowCreate: false,
+      },
+    },
+    {
+      name: 'treatments',
+      type: 'join',
+      collection: 'treatments',
+      on: 'tags',
+      admin: {
+        defaultColumns: ['name'],
+        description: 'Link this tag to one or more Treatments',
+        allowCreate: false,
+      },
```

Comment:
@ttothea this is the bi-directional approach. You have a join field on one side and a relationship on the other side. I read now that it is recommended to put the relationship field in these cases on posts, treatments and clinics when it is a N:M relationship rather than putting columns with relationships on the tags.

Nice thing is, that this field is virtual and does not create a column in the database.

Reply:
This is also the reason, why I have to change the ER diagram. I did it here not test it out, but afterwards we'll correct it with #84